pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/metrics/NcssMetric.java

```diff
@@ -41,7 +44,8 @@
 /**
  * Non Commenting Source Statements. Similar to LOC but only counts statements, which is roughly equivalent
  * to counting the number of semicolons and opening braces in the program. The precise rules for counting
- * statements comply with <a href="http://www.kclee.de/clemens/java/javancss/">JavaNCSS rules</a>.
+ * statements comply with <a href="http://www.kclee.de/clemens/java/javancss/">JavaNCSS rules</a>. The only
+ * difference is that import and package statements are not counted.
```

Comment:
In part yes. I think this exception makes it possible to compare nested classes to outer classes too.

But yes, it would be good to have version that complies fully with JavaNCSS! I'll work on that for the next PR